package.json

```diff
@@ -30,11 +30,7 @@
   "homepage": "https://github.com/gatsbyjs/gatsby-starter-landing-page#readme",
   "dependencies": {
     "gatsby": "^4.2.0",
-    "gatsby-plugin-image": "^2.2.0",
-    "gatsby-plugin-sharp": "^4.2.0",
-    "gatsby-source-filesystem": "^4.2.0",
     "gatsby-theme-landing-page": "*",
-    "gatsby-transformer-sharp": "^4.2.0",
```

Comment:
Since the theme includes these dependencies, the demo site doesn't need to install them